<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CRR22JH" failures="3" tests="6" name="edgetest" time="74.086" errors="0" timestamp="2024-11-09T12:30:02 IST">
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="9.437"/>
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="8.715">
    <failure type="java.lang.AssertionError" message="Error During Login: Invalid username/password. Please try again.">
      <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyLOgin -->
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="10.711"/>
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="7.7"/>
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="4.298">
    <failure type="java.lang.AssertionError" message="Error During Login: Invalid username/password. Please try again.">
      <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyLOgin -->
  <testcase classname="testCases.TC001_LoginPageTest" name="verifyLOgin" time="2.969">
    <failure type="java.lang.AssertionError" message="Error During Login: Invalid username/password. Please try again.">
      <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyLOgin -->
</testsuite> <!-- edgetest -->
